i18n-helpers/src/gettext.rs

```diff
@@ -321,6 +321,34 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_translate_html_in_table() {
+        let catalog = create_catalog(&[
+            ("Icon", "ICON"),
+            ("Description", "DESCRIPTION"),
+            (
+                "<img src=\"some-icon.svg\" alt=\"Some icon.\"/>",
+                "<img src=\"some-icon.svg\" alt=\"SOME ICON.\"/>",
+            ),
+            ("Some description.", "SOME DESCRIPTION."),
+        ]);
+        // The alignment is lost when we generate new Markdown.
+        assert_eq!(
+            translate(
+                "\
+                | Icon                                            | Description       |\n\
+                |-------------------------------------------------|-------------------|\n\
+                | <img src=\"some-icon.svg\" alt=\"Some icon.\"/> | Some description. |",
```

Comment:
Just an idea: consider "translating the `src` attribute and dropping the `alt` attribute to make the example smaller. I believe one attribute should be enough to demonstrate this.

Reply:
Sounds good! I just intended to write some practical example, but it isn't really necessary for testing.